engines/scumm/string.cpp

```diff
@@ -572,6 +573,31 @@ bool ScummEngine::newLine() {
 	return true;
 }
 
+#ifdef ENABLE_HE
+void ScummEngine_v72he::fakeBidiString(byte *ltext, bool ignoreVerb) const {
+	if (*ltext == 0x7F) {
+		ltext++;
+		while (*(ltext++) != 0x7F);
+	}
+	byte *loc = ltext;
+	byte tmp = 0;
+	while (1) {
+		while (*loc && *loc != 13) {
+			loc++;
```

Comment:
You could use forEachLine() inside Common::String to avoid all of this string manipulation per line

Reply:
Thanks,
tried look into it, but it seens the current implementation of `forEachLine` is not suitable for this as it
hard-coded to use `\n` (10) as new-line token
and require to use a `Common::String` object which might not be stable long-term for modifying the underlying buffer
(this implementation reverses the strings in-place)